WooCommerce/WooCommerceTests/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewModelTests.swift

```diff
@@ -108,3 +125,9 @@ final class BulkUpdateViewModelTests: XCTestCase {
         XCTAssertEqual(viewModel.syncState, .synced)
     }
 }
+
+private extension BulkUpdateViewModelTests {
+    private enum Localization {
+        static let priceTitle = NSLocalizedString("Price", comment: "The title for the price settings section in the product variation bulk update screen")
+    }
+}
```

Comment:
We could make the `Localization` enum in the VM have the default access level, but would I don't like to sacrifice code quality just for enabling testing. Since we do not have the final english transaction the best we can do is to check if it is empty. So I updated the code to do so!

Just a though about this, if we would like to test that we are using the correct localization key we could during the test action not include the `Localizable.strings` resulting in all localized macros to have as value their keys.